src/browsers/devicefarm.ts

```diff
@@ -13,34 +10,17 @@ export interface DevicefarmOptions {
   retryCount?: number;
   projectArn: string;
   expiresInSeconds?: number;
-  credentials: CredentialsOptions;
 }
 
 export default class DevicefarmBrowserCreator extends BrowserCreator {
   protected async __getBrowserUrl(): Promise<URL> {
     const options = this.options as DevicefarmOptions;
-    const client = new DeviceFarm({ credentials: options.credentials, region: 'us-west-2' });
+    const client = new DeviceFarm({ region: 'us-west-2' });
 
-    const response = await pRetry(
```

Comment:
Removed custom retry code, because aws-sdk has this built-in. Tested 50 parallel requests, all succeed.